src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -169,7 +169,7 @@ class AuthScreens extends React.Component {
 
     render() {
         // Wait to resolve initial Home route params.
-        if (!this.initialReportID) {
+        if (!_.isString(this.initialReportID)) {
             return null;
         }
```

Comment:
What I'm trying to say here: https://github.com/Expensify/Expensify.cash/pull/2575#pullrequestreview-644939666 is most of the issues would come when we return `null` here and there's no mounted navigator

The code would become simpler with `initialReportID` defaulting to empty string, and when no stored report is available during init we still mount the navigator, when the ReportScreen mounts it can make a call to `Navigation.navigate(/* any/first available report */)` if there's nothing available **and the screen is focused** - e.g. it would not mess navigation if we're deeplinked to some other screen

Reply:
That makes sense. But to back up for a second, wasn't the point of waiting so that we could pass the `initialReportID` to `initialParams` of the screen?

Reply:
Yes, but this waiting is causing issue like the one we're trying to solve. 

By removing it we'll won't have to come up with solutions like `try/catch` on certain `navigate` calls or custom nav handling like `navigateLater` 

Initial Prams was a good solution to fix the url in the address bar, but it's not the only one
I'll make sure to link the previous PR as well so that there's no regression on that matter